Adds a direct link back to the landing page

annegentle opened the pull request:
Fixes an issue noted here:
https://github.com/rackerlabs/docs-core-infra-user-guide/issues/408

annegentle committed:
Adds a direct link back to the landing page

Fixes an issue noted here:
https://github.com/rackerlabs/docs-core-infra-user-guide/issues/408

rackernexus commented:
:eyes: [Preview](https://staging.developer.rackspace.com/build-8c9974f280/docs/user-guides/orchestration/)

annegentle committed:
Adds to _toc.rst also.

Could you add a link to
https://developer.rackspace.com/docs/user-guides/orchestration/ for the repo?

rackernexus commented:
:eyes: [Preview](https://staging.developer.rackspace.com/build-376bed43b3/docs/user-guides/orchestration/)

annegentle commented:
Nope, not doing quite what I want. Will dig deeper.
